src/FSharp.Stats/Testing/MultipleTesting.fs

```diff
@@ -63,6 +63,16 @@ module MultipleTesting =
         |> Seq.ofList
 
 
+    // adapted from: https://en.wikipedia.org/wiki/%C5%A0id%C3%A1k_correction   &   https://personal.utdallas.edu/~herve/Abdi-Bonferroni2007-pretty.pdf
+    // Old: /// Computes the Dunn-Šidák correction onto a collection of p-values with a given alpha. Returns a list of booleans which state if the p-value should be rejected or not, and the new critical value for this alpha.
+    /// Computes the Dunn-Šidák correction onto a collection of p-values with a given alpha. Returns a critical value. p-values above the critical value should be rejected.
+    let dunnSidak alpha (pValues : seq<float>) = 
+        let m = float (Seq.length pValues)
+        let criticalValue = 1. - (1. - alpha) ** (1. / m)
+        // List.map (fun p -> if p > criticalValue then true, p else false, p) (List.ofSeq pValues),
```

Comment:
- remove this line
- What is the default procedure if pvalues in the input sequence are nan? Are they excluded from determining m? Should there be a special case? I don't know, but i can think of reasons why to exclude nan tests and maybe it is commonly performed.

Reply:
I did not find any case like this for this test. So I suggest to just exclude them. Same goes for values > 1.